cmd/node/config.go

```diff
@@ -50,3 +53,13 @@ func (cfg *NodeConfig) allFieldsPresent() error {
 	}
 	return nil
 }
+
+func (config *NodeConfig) generateNodeID() (string, error) {
+	data := []byte(config.Keys.Public)
```

Comment:
@stepanvalyavskiy I think it will be easier to reference file system path in config, instead of copying private and public keys here, WDYT?
E.g.:
```yaml
keys:
  public: /secret/key.pub
  private: /secret/key
```

Reply:
sure, it was so according to #27 
Fixed